README.md

```diff
@@ -6,15 +6,15 @@
 [![Coverage Status](https://coveralls.io/repos/github/fluentassertions/fluentassertions/badge.svg?branch=develop)](https://coveralls.io/github/fluentassertions/fluentassertions?branch=develop)
 
 # About this project
-A very extensive set of extension methods that allow you to more naturally specify the expected outcome of a TDD or BDD-style unit tests. Targets .NET Framework 4.7, as well as .NET Core 2.1, .NET Core 3.0, .NET Standard 2.0 and 2.1.
+A very extensive set of extension methods that allow you to more naturally specify the expected outcome of a TDD or BDD-style unit tests. Targets .NET Framework 4.7, as well as .NET Core 2.1, .NET Core 3.0, .NET 6, .NET Standard 2.0 and 2.1.
 
 See https://www.fluentassertions.com for [background information](https://fluentassertions.com/about/), [usage documentation](https://fluentassertions.com/introduction), an [extensibility guide](https://fluentassertions.com/extensibility/), support information and more [tips & tricks](https://fluentassertions.com/tips/).
 
 # Who created this?
 Originally authored by Dennis Doomen with Jonas Nyrup as the productive side-kick. Notable contributions were provided by Artur Krajewski, Lukas Grützmacher and David Omid.
 
 # How do I build this?
-Install Visual Studio 2019 16.9+ or JetBrains Rider 2021.1.0 as well as the Build Tools 2019 (including the Universal Windows Platform build tools). You will also need to have .NET Framework 4.7 SDK and .NET 5.0 SDK installed. Check [global.json](global.json) for the current minimum required version.
+Install Visual Studio 2022 17.0+ or JetBrains Rider 2021.3 as well as the Build Tools 2019 (including the Universal Windows Platform build tools). You will also need to have .NET Framework 4.7 SDK, .NET 5.0 SDK and .NET 6.0 SDK installed. Check [global.json](global.json) for the current minimum required version.
```

Comment:
🤔 Don't we need to bump the Build Tools to 2022 too?
🤔 Why do we need the .NET 5.0 SDK?

Reply:
_Technically_, you can install VS without the necessary .NET SDKs.

Reply:
Yes, but since you're no longer targeting .NET 5, you don't need the .NET 5 SDK. 
And if you're not installing Visual Studio (like me), you still need the 2022 Build Tools.

Reply:
Wow, _completely_ missed what you what you were saying.
You are absolutely right.